sarracen/disc/surface_density.py

```diff
@@ -284,6 +286,8 @@ def _calc_scale_height(data: 'SarracenDataFrame',
     H: Series
         The scale height of the disc.
     """
+
+    origin = _get_origin(origin)
```

Comment:
Instead of adding `_get_origin()`, we should make origin a mandatory argument instead of optional. Both places that call `_calc_scale_height()` already call `_get_origin()`, so I think we can avoid having default None here.

So remove the call to `_get_origin()` and change the argument to be just `origin: list`, without a default value (i.e., remove the = None).